docs/igor0806/GamesProject/scripts/games.js

```diff
@@ -0,0 +1,136 @@
+// games-mock.js
```

Comment:
Remove the comment, as you don't have any mocks

Reply:
This script looks so similar to the [script](https://github.com/MastersAcademy/fe-course-basic-2023/pull/443/files#diff-8892e7037f55b059dbec6b8d3181746d4cae02ddea10be9cac5e5d2c246b5e25R1) of your colleague.  If so, please be sure, that you understand, what is going on here.

It is up to you, the main aim of this task is to teach you something.